.idea/MaterialCalc.iml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We should exclude all the editor files to .gitignore

create a **.gitgnore** file and add `.idea` in it